pkg/tsdb/cloudwatch/cloudwatch.go

```diff
@@ -59,7 +59,7 @@ const (
 	timeSeriesQuery = "timeSeriesQuery"
 )
 
-var logger = log.New("tsdb.cloudwatch")
+var logger = log.New().With("logger", "tsdb.cloudwatch")
```

Comment:
This now logs things in `JSON` format instead of `Logfmt`. We'll need to move the instantiation of the logger to be inside `ProvideService`.

You can see an example of this done in these 2 PRs: https://github.com/grafana/grafana/pull/79147 and https://github.com/grafana/grafana/pull/79294

Reply:
Fascinating! I'm not sure _how_ that change works, but I did something similar here (and tested this time 😅) and it sure does work. It also cleans up the code a fair bit, not passing the logger around everywhere. Thanks!